src/character.cpp

```diff
@@ -230,6 +230,16 @@ void Character::mod_stat( const std::string &stat, float modifier )
     }
 }
 
+int Character::get_fat_to_hp() const
+{
+    int mut_fat_hp = 0;
+    for( const trait_id &mut : get_mutations() ) {
+        mut_fat_hp += mut.obj().fat_to_max_hp;
```

Comment:
maybe this could be a float instead of an int? i'd certainly like the additional options that would afford.